docs/estela/installation/helm-variables.md

```diff
@@ -71,6 +71,9 @@ you have a deep understanding of estela.
 * _<SPIDERDATA\_DB\_ENGINE>_ (Required): Document oriented database where the data produced 
   by the spiders is stored. Currently, estela supports the _mongodb_ engine.
 
+  > **NOTE:**
+  >  For dev deploy  [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) can be used to set a database, as mentioned on [Estela Resources Guide](./resources.md)
```

Comment:
MongoDB Atlas can used, although the Usage Records will not work properly since MongoDB Atlas does not allow using the commands to get the database and collections sizes which are required by the Usage Records.

Reply:
so we will need to provide a local dev option, because MongoDB Atlas is the only one mentioned on [Estela Resources Guide](./resources.md)

Reply:
I think you can add a note regarding set a local mongoDB cluster or MongoAtlas(limited by commands not allowed in free trial)

Reply:
done